### ISSUE TITLE
Add the Bitergia user/bot to Slack Channels

### ISSUE THREAD

**Issue by e0d** [openedx/axim-engineering#275]
Bitergia needs to subscribe owlbot@bitergia.com to channels that need to be tracked and install the OwlBot-app app.

The Admin should also give us permission to create tokens, or he could send a token for us. [Detailed Instructions](https://gitlab.com/Bitergia/c/FAQ/-/blob/master/how-to-sources/README.md#how-to-enable-the-tracking-of-a-slack-instance).

**Comment by feanil** [comment]
The ticket is marked as done on the tCRIL board, should this also get closed?

**Comment by sarina** [comment]
I don't think we ever got a Bitergia bot integration, but I'm not sure what this would get us. Regardless as Bitergia is shutting down I'm closing this ticket as a no-op
